src/labels/CustomLabels.labels

```diff
@@ -7328,6 +7328,182 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
         <shortDescription>Title of checklist item FundraisingProcess in Get Started</shortDescription>
         <value>Visualize Your Fundraising Processes</value>
     </labels>
+    <labels>
+        <fullName>gseuChecklistItemCaptureDonorIntentDesc</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of checklist Item Capture Donor Intent in GS for End User</shortDescription>
+        <value>When you raise funds for a specific purpose, you want to be sure you are honoring the donor intent. Ensure donations are allocated correctly with General Allocation Units.</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemCaptureDonorIntentLinkLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Link label of checklist Item Capture Donor Intent in GS for End User</shortDescription>
+        <value>Learn more about Donation Allocations in Trailhead</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemCaptureDonorIntentPriBtnLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Primary button label of checklist Item Capture Donor Intent in GS for End User</shortDescription>
+        <value>Create Allocations</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemCaptureDonorIntentTitle</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of checklist Item Capture Donor Intent in GS for End User</shortDescription>
```

Comment:
I recommend changing this to: Description of Capture Donor Intent checklist item in Get Started for End Users.